ingestion/src/metadata/pii/ner_scanner.py

```diff
@@ -48,20 +48,23 @@ class NERScanner:
     Based on https://microsoft.github.io/presidio/
     """
 
-    import spacy
-    from presidio_analyzer import AnalyzerEngine
-    from presidio_analyzer.nlp_engine.spacy_nlp_engine import SpacyNlpEngine
+    def __init__(self):
+        import spacy
+        from presidio_analyzer import AnalyzerEngine
+        from presidio_analyzer.nlp_engine.spacy_nlp_engine import SpacyNlpEngine
 
-    try:
-        spacy.load(SPACY_EN_MODEL)
-    except OSError:
-        logger.warning("Downloading en_core_web_md language model for the spaCy")
-        from spacy.cli import download
+        try:
+            spacy.load(SPACY_EN_MODEL)
+        except OSError:
+            logger.warning("Downloading en_core_web_md language model for the spaCy")
+            from spacy.cli import download
 
-        download(SPACY_EN_MODEL)
-        spacy.load(SPACY_EN_MODEL)
+            download(SPACY_EN_MODEL)
```

Comment:
Here if the download fails for some reason, the entire workflow will fail. Might be better to handle the exception

Reply:
I think I am ok on making such errors super evident. Otherwise it might be confusing to have silent download failures and just skip the auto PII. If the errors persists or cannot be fixed for whatever reason, users can disable the tagging and move on.

wdyt?

Reply:
Ok it should be fine then if the user can disable it if it fails 👍